core/src/main/java/hudson/slaves/Cloud.java

```diff
@@ -81,7 +83,7 @@
  * @see NodeProvisioner
  * @see AbstractCloudImpl
  */
-public abstract class Cloud extends AbstractModelObject implements ExtensionPoint, Describable<Cloud>, AccessControlled {
+public abstract class Cloud extends Actionable implements ExtensionPoint, Describable<Cloud>, AccessControlled {
```

Comment:
yep. The ones introduced by Actionable are not mandatory though it would not hurt